[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_2_receiving_nodes__relay_node1_forwards_lightpushed_message_to_relay_node2 with id: 2024-07-03_04-14-16__ea5131c1-1bb7-4827-bd9c-519deb45fac6
[35mDEBUG   [0m src.steps.common:common.py:17 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.light_push:light_push.py:27 Running fixture setup: light_push_setup
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:17 Docker client initialized with image harbor.status.im/wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:80 WakuNode instance initialized with log path ./log/docker/receiving_node1_2024-07-03_04-14-16__ea5131c1-1bb7-4827-bd9c-519deb45fac6__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:84 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:20 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:23 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:80 Generated random external IP 172.18.86.197
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:73 Generated ports ['27876', '27877', '27878', '27879', '27880']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:426 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:155 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:157 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:47 docker run -i -t -p 27876:27876 -p 27877:27877 -p 27878:27878 -p 27879:27879 -p 27880:27880 harbor.status.im/wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=27878 --rest-port=27876 --tcp-port=27877 --discv5-udp-port=27879 --rest-address=0.0.0.0 --nat=extip:172.18.86.197 --peer-exchange=true --discv5-discovery=true --cluster-id=0 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=27880 --metrics-logging=true --lightpush=true --relay=true --nodekey=30348dd51465150e04a5d9d932c72864c8967f806cce60b5d26afeca1e77eb68
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:53 docker network connect --ip 172.18.86.197 waku 4b828442c607b553dedfb9196a32faad388590ac452ad6de9ac5eb8792190b77
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:56 Container started with ID 4b828442c607. Setting up logs at ./log/docker/receiving_node1_2024-07-03_04-14-16__ea5131c1-1bb7-4827-bd9c-519deb45fac6__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:169 Started container from image harbor.status.im/wakuorg/nwaku:latest. REST: 27876
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:27876/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:272 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:27876/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.86.197/tcp/27877/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip","/ip4/172.18.86.197/tcp/27878/ws/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip"],"enrUri":"enr:-LO4QHOrZfLDdXG7My3RtXfarmP9Xv7rotzxTXHY6HFpGomZWS6l579jsg8VYuuUhVlXA0UxXZDH9fb_gYDXQN8KcR0BgmlkgnY0gmlwhKwSVsWKbXVsdGlhZGRyc4wACgSsElbFBmzm3QOCcnOFAAABAACJc2VjcDI1NmsxoQM3Tqpf5eFn4Jztm4gB0Y0JVSJyxyZsW8QR-QU5DZb-PYN0Y3CCbOWDdWRwgmznhXdha3UyCQ"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 REST service is ready !!
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:17 Docker client initialized with image harbor.status.im/wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:80 WakuNode instance initialized with log path ./log/docker/receiving_node2_2024-07-03_04-14-16__ea5131c1-1bb7-4827-bd9c-519deb45fac6__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:84 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:20 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:23 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:80 Generated random external IP 172.18.169.202
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:73 Generated ports ['48981', '48982', '48983', '48984', '48985']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:426 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:155 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:157 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:47 docker run -i -t -p 48981:48981 -p 48982:48982 -p 48983:48983 -p 48984:48984 -p 48985:48985 harbor.status.im/wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=48983 --rest-port=48981 --tcp-port=48982 --discv5-udp-port=48984 --rest-address=0.0.0.0 --nat=extip:172.18.169.202 --peer-exchange=true --discv5-discovery=true --cluster-id=0 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=48985 --metrics-logging=true --lightpush=false --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:53 docker network connect --ip 172.18.169.202 waku 6df71987f4ac32c31a9249da243812f5dccc4bdfed3c3be89fba55977fc5a23a
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:56 Container started with ID 6df71987f4ac. Setting up logs at ./log/docker/receiving_node2_2024-07-03_04-14-16__ea5131c1-1bb7-4827-bd9c-519deb45fac6__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:169 Started container from image harbor.status.im/wakuorg/nwaku:latest. REST: 48981
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:48981/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:272 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:48981/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.169.202/tcp/48982/p2p/16Uiu2HAmF1H1BFJnJJHmwrEM3Z5emJadzyUbqr5X7ok5dxrM589Q","/ip4/172.18.169.202/tcp/48983/ws/p2p/16Uiu2HAmF1H1BFJnJJHmwrEM3Z5emJadzyUbqr5X7ok5dxrM589Q"],"enrUri":"enr:-LO4QDVDl3YtmY3BoEYrqWiEtjCWmfMhg-Y6rYo0WBC6jVkSRSe-a-7IYWQHMHZjzsA4AT-J7FrYTeUM3zlkCd9wG88BgmlkgnY0gmlwhKwSqcqKbXVsdGlhZGRyc4wACgSsEqnKBr9X3QOCcnOFAAABAACJc2VjcDI1NmsxoQMi6dHlZipb5dc4p5OzFKStoFbDmHWdGOrHiAYSGQ9LI4N0Y3CCv1aDdWRwgr9YhXdha3UyAQ"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:48981/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.86.197/tcp/27877/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:17 Docker client initialized with image harbor.status.im/wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:80 WakuNode instance initialized with log path ./log/docker/lightpush_node1_2024-07-03_04-14-16__ea5131c1-1bb7-4827-bd9c-519deb45fac6__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:84 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:20 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:23 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:80 Generated random external IP 172.18.253.22
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:73 Generated ports ['63528', '63529', '63530', '63531', '63532']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:426 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:155 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:157 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:47 docker run -i -t -p 63528:63528 -p 63529:63529 -p 63530:63530 -p 63531:63531 -p 63532:63532 harbor.status.im/wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=63530 --rest-port=63528 --tcp-port=63529 --discv5-udp-port=63531 --rest-address=0.0.0.0 --nat=extip:172.18.253.22 --peer-exchange=true --discv5-discovery=true --cluster-id=0 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=63532 --metrics-logging=true --discv5-bootstrap-node=enr:-LO4QHOrZfLDdXG7My3RtXfarmP9Xv7rotzxTXHY6HFpGomZWS6l579jsg8VYuuUhVlXA0UxXZDH9fb_gYDXQN8KcR0BgmlkgnY0gmlwhKwSVsWKbXVsdGlhZGRyc4wACgSsElbFBmzm3QOCcnOFAAABAACJc2VjcDI1NmsxoQM3Tqpf5eFn4Jztm4gB0Y0JVSJyxyZsW8QR-QU5DZb-PYN0Y3CCbOWDdWRwgmznhXdha3UyCQ --lightpushnode=/ip4/172.18.86.197/tcp/27877/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip --lightpush=true --relay=false
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:53 docker network connect --ip 172.18.253.22 waku cd0f977d9d7ac77d4c2696cee99e1067ccc51235c7e99f237452f035c30f9db2
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:56 Container started with ID cd0f977d9d7a. Setting up logs at ./log/docker/lightpush_node1_2024-07-03_04-14-16__ea5131c1-1bb7-4827-bd9c-519deb45fac6__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:169 Started container from image harbor.status.im/wakuorg/nwaku:latest. REST: 63528
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:63528/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:272 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:63528/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.253.22/tcp/63529/p2p/16Uiu2HAmPaXc2BTu7koiVEVhNtHAePJ9cDbFRvL64BY5fTfcVc1z","/ip4/172.18.253.22/tcp/63530/ws/p2p/16Uiu2HAmPaXc2BTu7koiVEVhNtHAePJ9cDbFRvL64BY5fTfcVc1z"],"enrUri":"enr:-LO4QBMRvOQdHOxvvQv5Eu7NcNmcjY4CCGZoDY8kjjXHrFk9Yc6zt0Ytl44QIAvOAFrDnKrJra7b4-R2M9JjiTu8wuYBgmlkgnY0gmlwhKwS_RaKbXVsdGlhZGRyc4wACgSsEv0WBvgq3QOCcnOFAAABAACJc2VjcDI1NmsxoQOiS_27bW31lorpLKGLxrI1rqFMecQTz923sxR-lPaS9YN0Y3CC-CmDdWRwgvgrhXdha3UyCA"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:63528/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.86.197/tcp/27877/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:63528/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.169.202/tcp/48982/p2p/16Uiu2HAmF1H1BFJnJJHmwrEM3Z5emJadzyUbqr5X7ok5dxrM589Q"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:27876/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/0/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:48981/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/0/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.light_push:light_push.py:120 Lightpushing message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:63528/lightpush/v1/message" -H "Content-Type: application/json" -d '{"pubsubTopic": "/waku/2/rs/0/0", "message": {"payload": "TGlnaHQgcHVzaCB3b3JrcyEh", "contentTopic": "/myapp/1/latest/proto", "timestamp": 1719980060836665856}}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_1:harbor.status.im/wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:27876/relay/v1/messages/%2Fwaku%2F2%2Frs%2F0%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1719980060836665856,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_2:harbor.status.im/wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:48981/relay/v1/messages/%2Fwaku%2F2%2Frs%2F0%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1719980060836665856,"ephemeral":false}]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:216 Stopping container with id 4b828442c607
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:216 Stopping container with id 6df71987f4ac
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:216 Stopping container with id cd0f977d9d7a
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Container stopped.